distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso

```diff
@@ -12,14 +12,17 @@ polyglot java import org.enso.table.error.EmptySheetException
 type Missing_Input_Columns
     ## PRIVATE
        One or more columns not found in the input table.
-    Error (criteria : [Text])
+    Error (criteria : [Text]) (where:Text|Nothing = Nothing)
```

Comment:
What is meant to go in 'where'?

Reply:
A location description, dependent on the particular operation. I will add a comment in subsequent PR.